acceptance-tests/helpers/services/create.go

```diff
@@ -22,16 +22,17 @@ type config struct {
 	name              string
 	serviceBrokerName func() string
 	parameters        string
+	plan string
 }
 
 type Option func(*config)
 
-func CreateInstance(offering, plan string, opts ...Option) *ServiceInstance {
-	cfg := defaultConfig(offering, plan, opts...)
+func CreateInstance(offering string, opts ...Option) *ServiceInstance {
```

Comment:
What's the rationale for making `plan` an option? As far as I can see it is required, so it makes the API slightly less usable because it's possible to forget a mandatory parameter. I think the experience could be improved by failing fast if there no plan is set with a helpful error message.

Reply:
Oh yeah this doesn't make sense in this PR because it was triggered by the DynamoDB test ( which I didn't include since I wanted to run it locally first)

The update can include plan and param updates and that's what triggered me to put the plan as an option even though it's less intuitive for the create flow. I'll add the error messaging you mentioned - that a good idea!